router/router.go

```diff
@@ -1491,6 +1490,10 @@ func (worker *workerT) canBackoff(job *jobsdb.JobT, userID string) (shouldBackof
 	return false
 }
 
+func (rt *HandleT) getWorkerPartition(userID string) int {
+	return int(math.Abs(float64(misc.GetHash(userID) % rt.noOfWorkers)))
```

Comment:
[question] why do we need abs here? Can `misc.GetHash(userID) % rt.noOfWorkers` be possibly negative ?

Reply:
you are correct, the hash actually return an unsigned integer...